modules/system/effect-wfrp4e.js

```diff
@@ -126,11 +126,15 @@ export default class EffectWfrp4e extends ActiveEffect {
     return this.parent?.type == "trapping" && getProperty(this, "flags.wfrp4e.reduceQuantity")
   }
 
-  reduceItemQuantity() {
+  get isAsync () {
+    return getProperty(this, "flags.wfrp4e.isAsync") && (game.wfrp4e.config.syncEffectTriggers.indexOf(this.trigger) === -1)
+  }
+
+  async reduceItemQuantity() {
     if (this.reduceQuantity && this.item)
     {
       if (this.item.quantity.value > 0)
-        this.item.update({"system.quantity.value" : this.item.quantity.value - 1})
+        await this.item.update({"system.quantity.value" : this.item.quantity.value - 1})
```

Comment:
Doesn't seem necessary to await this

Reply:
its only for someone that clicks too fast - it can then update the quantity incorrectly